test/autorest/generated/headergroup/internal/headergroup/header.go

```diff
@@ -0,0 +1,488 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package headergroup
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
+	"time"
+)
+
+type HeaderOperations struct{}
+
+// CustomRequestIDCreateRequest creates the CustomRequestID request.
+func (HeaderOperations) CustomRequestIDCreateRequest(u url.URL) (*azcore.Request, error) {
```

Comment:
Is this missing a param?

Reply:
That's the one with the special params that aren't specified in the swagger. Wasn't sure when to add to those cases.

Reply:
Ah ok.  We can work up a design for that case a little later, can you please open an issue to track it?

Reply:
Yep, here it is: #268